lib/FetchEnvs.js

```diff
@@ -227,6 +228,24 @@ module.exports = class FetchEnvs {
     const env = objectPath.get(this.data, `object.${path}.env`, []);
     return (await this.#processEnv(env)).reduce(reduceEnv, result);
   }
+
+  async getHeaders(requestOptions) {
```

Comment:
instead of putting http request logic into fetchEnvs, is it possible to expose `processEnvFrom` instead of making it a private function? Then we can just put this logic in the baseDownloadController